inst/get-testdata/score_untrained-mean.R

```diff
@@ -0,0 +1,186 @@
+## Data for testing the score_untrained example: mean output
+
+## target data
+target_data_mean <- readRDS(
```

Comment:
I can't just run this code as it requires `dplyr` at least. can you add the required libraries at the top of the file so this script stands on its own?